src/lib/server.ts

```diff
@@ -77,13 +80,13 @@ class Client {
       }
 
       throw new UnknownCommandError(msg.command);
-    } catch (err: unknown) {
+    } catch (err) {
```

Comment:
This will implies error is `any` while we do not know the type. I would leave it as `unknown`.